src/components/Molecules/PostcodeLookup/PostcodeLookup.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isValid, toNormalised } from 'postcode';
+import axios from 'axios';
+import Lookup from '../Lookup/Lookup';
+import StyledWrapper from './PostcodeLookup.style';
+
+const validatePostcode = postcode => {
+  const trimmed = typeof postcode === 'string' ? postcode.trim() : '';
+  return isValid(trimmed) && toNormalised(trimmed);
+};
+
+const getAddresses = postcode => axios.get(
+  'https://lookups-staging.sls.comicrelief.com/postcode/lookup',
+  {
+    timeout: 10000,
+    params: { query: postcode }
+  }
+).then(res => res.data.addresses || []);
+
+const addressToString = address => [address.Line1, address.Line2, address.Line3, address.posttown]
+  .filter(line => line)
+  .join(', ');
+
+const addressFetcher = async postcode => {
+  const valid = validatePostcode(postcode);
+  if (!valid) {
+    throw new Error('Please provide a valid postcode');
+  }
+  try {
+    return await getAddresses(valid);
+  } catch (error) {
+    // if (typeof Sentry !== 'undefined') {
```

Comment:
We shouldn't remove this, or we should at least provide a callback prop function that is called here, so we're able to run that Sentry function within the application this'll be used within.